app/views/assets/_edit_white.html.erb

```diff
@@ -28,7 +28,7 @@
     <% if @allow_reupload %>
 
     <div class="track-edit-bottom">
-      <%= link_to 'Upload a new version of this track',"new_version_#{asset.object_id}", class: 'slide_open_next'%>
+      <%= link_to 'Upload a new version of this track',"#new_version_#{asset.object_id}", class: 'slide_open_next'%>
```

Comment:
```suggestion
      <%= link_to 'Upload a new version of this track',"#new_version_#{asset.object_id}", class: 'slide_open_href'%>
```

This would solve the issue without breaking profile editing, I think.

Reply:
Good idea. I looked over current usage, and for some reason decided that it's not used... I'll fix it.